menu/cakes/cakeB.html

```diff
@@ -0,0 +1,250 @@
+  <link rel="stylesheet" href="../../assets/css/main.css" />
+  <link rel="icon" type="image/x-icon" href="../../images/favicon.ico" />
+</head>
+
+<body class="is-preload">
+
+  <!-- Wrapper -->
+  <div id="wrapper">
+
+    <!-- Main -->
+    <div id="main">
+      <div class="inner">
+
+        <!-- Header -->
+        <header id="header">
+          <a href="../../index.html" class="logo"><strong>Mandy B. Bakes</strong></a>
+          <ul class="icons">
+           <li><a href="https://instagram.com/mandyb.bakes" class="icon brands fa-instagram"><span
+                  class="label">Instagram</span></a></li>
+          </ul>
+        </header>
+
+        <!-- Banner -->
+        <section id="banner">
+          <span class="image image-left object">
+            <img src="../../images/cakes/cake_pic04.jpg"
+                  alt="kid's rodeo themed birthday cake" />
+          </span>
+          <div class="content content-right item-details">
+            <header>
+              <h1>Cake B</h1>
+            </header>
+            <h2>Description:</h2>
+            <p>Aenean ornare velit lacus, ac varius enim ullamcorper eu. Proin aliquam facilisis ante interdum congue.
+              Integer mollis, nisl amet convallis, porttitor magna ullamcorper, amet egestas mauris. Ut magna finibus
+              nisi nec lacinia. Nam maximus erat id euismod egestas. Pellentesque sapien ac quam. Lorem ipsum dolor sit
+              nullam.</p>
+            <form class="orderForm" action="https://formspree.io/f/xvoepdpy" method="POST">
+            
+              <div>
+                <h3>Select Your Options:</h3>
+                
+                <!-- Pass Cake Name to Form -->
+                <label class="hidden">
+                  <input type="text" name="itemName" value="Cake A" readonly>
+                </label>
+
+                <label class="hidden">
+                  What would you like?
+                  <select class="cakeOrCupcakes" name="cake or cupcakes">
+                    <option value="">- Choose one -</option>
+                    <option value="cake" selected>Cake</option>
+                    <option value="cupcakes">Cupcakes</option>
+                  </select>
+                </label>
+            
+                <label class="cakeSize">
+                  Size of the cake
+                  <select class="cakeSizeSelect" name="cake size">
+                    <option value="">- Choose one -</option>
+                    <option value="4in">4-inch (8 servings)</option>
+                    <option value="6in">6-inch (12 servings)</option>
+                    <option value="8in">8-inch (24 servings)</option>
+                    <option value="10in">10-inch (34 servings)</option>
+                  </select>
+                </label>
+            
+                <label>
+                  When do you need the cake by?
+                  <input type="date" name="date" id="orderNeededDate" required>
```

Comment:
we may need to break orderNeededDate into 2 util functions, one for 1 month away from now, and one for 1 week or something away. We will let Amanda decide but baking a pie should not take as much time as preparing for a wedding cake.

Reply:
good catch! I will work on breaking out orderNeededDate into 2 util functions.

Reply:
Confirming I finally split this out into 2 util functions. I decided to add a class called "shortLead" to the non-wedding products in addition to the id "orderNeededDate". I added the class to keep the id the same regardless of lead time and the id name was already long. 
Also, I thought it could give us some flexibility vs naming the class "weekLead" or "monthLead"  in case the lead times need to change, we can adjust that within the util function (while not needing to adjust within the individual products).